KiwiBoard.emf

```diff
@@ -375,7 +375,7 @@
     {
       "parentId": 15,
       "type": "analogItem",
-      "defaultValue": "4",
+      "defaultValue": "false",
```

Comment:
This was likely a merge error, default value should probably go back to 4

Reply:
I don't recall adjusting this